### PR TITLE
Add constant padding to random hexes, hashes and addresses

### PR DESCRIPTION
**Pull request by barrutko**

### CODE REVIEW

File: packages/common-universal/src/types/CheckedAddress.ts [hunk 1]
```diff
@@ -19,10 +19,11 @@ export function CheckedAddress(value: string): CheckedAddress {
  * Generates a random address. Tries to represent desired ascii prefix as hex value. Helps to identify transactions in the logs.
  */
 CheckedAddress.random = (asciiPrefix = ''): CheckedAddress => {
+  const constantAddressPrefix = '0000'
```
Comment (krzkaczor) [inline packages/common-universal/src/types/CheckedAddress.ts:22]: i thought about this more: i think a good prefix is `deadbeef` as it's obviously not random

Comment (barrutko) [reply packages/common-universal/src/types/CheckedAddress.ts]: changed to deadbeef

File: packages/common-universal/src/types/CheckedAddress.ts [hunk 2]
```diff
@@ -19,10 +19,11 @@ export function CheckedAddress(value: string): CheckedAddress {
  * Generates a random address. Tries to represent desired ascii prefix as hex value. Helps to identify transactions in the logs.
  */
 CheckedAddress.random = (asciiPrefix = ''): CheckedAddress => {
```
Comment (oskarvu) [inline packages/common-universal/src/types/CheckedAddress.ts:21]: I guess it will be `infix` now, since it's not first part of the address anymore.

I'm curious, why this change was needed in the first place?

Comment (barrutko) [reply packages/common-universal/src/types/CheckedAddress.ts:21]: I don't know if we need to be that precise to rename that to `infix` - both of these parts are still `pre`randomPart :P

**Review by yivlad — APPROVED** [review]

**Review by oskarvu — APPROVED** [review]
